### DISCUSSION TITLE
Enable deployments of data products to other regions besides data landing zone region

### DISCUSSION THREAD

**Discussion by renepajta** [Azure/data-domain-batch#77] [Ideas] [answered]
At the moment, data product can be deployed only to data landing zone region due to Synapse workspace limitations (storage account must be in the same region as Synapse workspace).

![image](https://user-images.githubusercontent.com/6043628/127906251-ae5b2da5-50f3-43d8-ae12-edc9dd55c0d4.png)
![image](https://user-images.githubusercontent.com/6043628/127906213-d1604087-cca9-4484-b75f-cc088968bc75.png)

This would imply a hard criteria to deploy data landing zone per region. 

There might be cases when you would like to work on data from the same domain/department but build your product in other region. 

Would you see this as a case for separate DLZ or a feature request for data product?

**Comment by marvinbuss** [comment, marked as answer]
Thanks  @renepajta for the feedback. Yes, today we are proposing to have a Landing Zone per region in which you want to store and process data. Could you please elaborate on why this might be different?

* Why would you want to create a Data Product in a different region and not n the region where your data is stored?
* Why do you want to host compute in a separate region than your data?
* Can you not just host the results of the analysis in a different region if necessary?

**Reply by renepajta** [reply]
I have been mainly confused by the introduction session where we gave pretty much vague definition for separation between data landing zones (e.g. per region, department, function) . I need to re-read the docs to avoid confusion.

Your questions are on point and in most cases we won't need data products in other region. 

I have a use case with requirements to provide regional DWH as a product with consistent development and overlapping data sources, so based on the above information I could imagine potential solution:
* Deploy regional DWH per DLZ
  * Deploy a DLZ per region
  * Create a Data Product template for DWH
  * Deploy regional DWH as a product to each respective DLZ
  * Share common code base  (ADF ETL, Synapse structuce, etc.) via a repository
  * Request necessary data from other regional DWH (most likely its a subset of data)

Thank you @marvinbuss  for guidance and great hints.

**Reply by marvinbuss** [reply]
Thanks for the feedback. Let's discuss the Data Product for DWs. Maybe we can support here and collaborate, so that others can also make use of this. Alternatively, maybe one of our existing products can be used for this (data-product-batch)?!

For the data share between DWs, we might be able to use a native Azure feature soon. Can share more with you in a separate thread. Besides that, I agree with the points you are mentioning above.
